src/components/Pagination/Pagination.test.tsx

```diff
@@ -0,0 +1,434 @@
+import { fireEvent, render, RenderResult, screen } from '@testing-library/react';
+import React, { ReactNode } from 'react';
+
+import { documentOf } from '../../helpers/document-of';
+
+import Pagination from './Pagination';
+
+const renderPagination = ({ pageCount = 10, ...rest }): RenderResult => {
+	return render(<Pagination pageCount={pageCount} {...rest} />);
+};
+
+const renderButton = (label: string): ReactNode => {
+	return <span>{label}</span>;
+};
+
+describe('<Pagination />', () => {
+	it('Should be able to render', () => {
+		const rendered = renderPagination({});
+
+		const pagination = documentOf(rendered).getElementsByClassName('c-pagination');
```

Comment:
Uw container is normaal uw root element en daar kan je gwn een `.querySelector()` op uitvoeren.
Heb je die `documentOf` dan nog nodig? Idem voor alle andere waar je `getElementsByClassName` gebruikt.

Reply:
Dit staat in de docs van `react-testing-library`, maar misschien doe ik hier gewoon hetzelfde op een andere manier 🤔 
![image](https://user-images.githubusercontent.com/85545258/147925297-b3dc24a1-66af-42e8-813c-abe5f877fa46.png)